packages/account-v2/src/components/FormFields/__tests__/FormDropDownField.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { useBreakpoint } from '@deriv/quill-design';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import FormDropDownField from '../FormDropDownField';
+
+jest.mock('@deriv/quill-design', () => ({
+    useBreakpoint: jest.fn(() => ({ isMobile: false })),
+}));
+
+describe('FormDropDownField', () => {
+    it('should render the dropdown field', () => {
+        render(
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            <Formik initialValues={{}} onSubmit={() => {}}>
```

Comment:
Can we assign `jest.fn` to onSubmit rather than an empty function?

Reply:
Updated.